BrainPortal/app/models/cluster_task.rb

```diff
@@ -139,7 +139,7 @@ def save_results
   # job on the cluster. The value returned by a
   # CbrainTask should be conservative and be reasonably
   # larger than the longest run expected, without being
-  # overly excessive. The default value used by
+  # overlay excessive. The default value used by
```

Comment:
This comment change looks like a typo to me. "overlay excessive" doesn't make grammatical sense.